Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/Validation/EconDamperPositionLimitsMultiZone_VOut_flow.mo

```diff
@@ -0,0 +1,86 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.AtomicSequences.Validation;
+model EconDamperPositionLimitsMultiZone_VOut_flow
+  "Validation model for the multiple zone VAV AHU minimum outdoor air control - damper position limits"
+  extends Modelica.Icons.Example;
+
+  parameter Real minVOut_flowSet(unit="m3/s")=0.71
+    "Example volumetric airflow setpoint, 15cfm/occupant, 100 occupants";
+  parameter Real minVOut_flow(unit="m3/s")=0.61
+    "Minimal measured volumetric airflow";
+  parameter Real VOut_flowIncrease(unit="m3/s")=0.2
+    "Maximum volumetric airflow increase during the example simulation";
+
+  CDL.Continuous.Constant VOut_flowMinSet(k=minVOut_flowSet)
+    "Outdoor volumetric airflow rate setpoint, 15cfm/occupant and 100 occupants"
+    annotation (Placement(transformation(extent={{-60,30},{-40,50}})));
+  Modelica.Blocks.Sources.Ramp VOut_flow(
+    duration=1800,
+    offset=minVOut_flow,
+    height=VOut_flowIncrease)
+    "Measured outdoor airflow rate"
+    annotation (Placement(transformation(extent={{-60,70},{-40,90}})));
+
+  EconDamperPositionLimitsMultiZone ecoDamLim
+    "Multiple zone VAV AHU minimum outdoor air control - damper position limits"
+    annotation (Placement(transformation(extent={{20,-10},{40,10}})));
+
+protected
+  parameter Types.FreezeProtectionStage freProDisabled = Types.FreezeProtectionStage.stage0
+    "Indicates that the freeze protection is disabled";
+  parameter Integer freProDisabledNum = Integer(freProDisabled)-1
+    "Numerical value for freeze protection stage 0 (=0)";
+  parameter Types.OperationMode occupied = Types.OperationMode.occupied
+    "Operation mode is \"Occupied\"";
+  parameter Integer occupiedNum = Integer(occupied)
+    "Numerical value for \"Occupied\" operation mode (=1)";
```

Comment:
Don't use quotes (we also don't use it in other models, and you don't use it on line 41 either.